scripts/tests.bat

```diff
@@ -31,7 +31,7 @@ set ETHEREUM_DEPS_PATH=%4
 if "%TESTS%"=="On" (
 
     cd ..
-    git clone https://github.com/ethereum/tests.git
+    git clone --depth 1 https://github.com/ethereum/tests.git
```

Comment:
This is about history depth. It just downloads more/less latest commit. Nothing but optimization.